src/framework/auth/providers/dummy-auth.provider.ts

```diff
@@ -2,8 +2,9 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 import { of as observableOf } from 'rxjs/observable/of';
 
-import { NbAuthResult } from '../services/auth.service';
+import { NbAuthResult } from '../services/auth-result';
 import { NbAbstractAuthProvider } from './abstract-auth.provider';
+import { delay } from 'rxjs/operators/delay';
```

Comment:
let's order imports like this
- angular imports
- 3rd party imports
- our local imports